testModule/settings.gradle.kts

```diff
@@ -1,23 +1,13 @@
-import com.github.javaparser.printer.concretesyntaxmodel.CsmElement.attribute
-import groovy.xml.dom.DOMCategory.attributes
-
-pluginManagement {
-	repositories {
-		google()
-		gradlePluginPortal()
-		mavenCentral()
-
-		mavenLocal()
-	}
-
-}
-
 buildscript {
-	repositories {
-		google()
-		gradlePluginPortal()
-		mavenCentral()
-
-		mavenLocal()
-	}
+    pluginManagement {
+        repositories {
+            gradlePluginPortal()
+            maven("https://repo.sleeping.town") {
+                content {
+                    includeGroup("com.unascribed")
+                }
+            }
+        }
+    }
 }
+includeBuild("../")
```

Comment:
What's this do?

Reply:
It makes the test module include the plugin, so you don't need to publish it to maven local first